docs/data/common-concepts/custom-components/custom-components.md

```diff
@@ -0,0 +1,244 @@
+# Custom slots and subcomponents
+
+<p class="description">Learn how to override the default DOM structure of the MUI X components.</p>
```

Comment:
```suggestion
<p class="description">Learn how to override parts of the MUI X components.</p>
```

"override the default DOM structure" feels a bit misleading to me.

Reply:
It was the wording on the pickers page
But I also prefer your version